### DISCUSSION TITLE
Rebus and Health Checks

### DISCUSSION THREAD

**Discussion by ionutvasiliu** [rebus-org/Rebus#1073] [Q&A]
Hi all,

I have a NET 6 console application using Rebus. Everything works perfectly. But then I decided to add some ASP NET Core health checks to it. None of them were working until I commented out the Rebus configuration section.

Their default port is 5000. Is there something in Rebus that uses this port?

Thanks

**Comment by mookid8000** [comment]
> Their default port is 5000. Is there something in Rebus that uses this port?

There's nothing in Rebus that will just open a socket on port 5000.

Could you maybe show some more about what you tried to do? E.g. what does your Rebus configuration look like, which section did you comment out to make it work, etc.?

**Comment by mookid8000** [comment]
Closing - please don't hesitate to get back in here, if you want to talk some more about this. 🙂
